### PR TITLE
KFSPTS-5876 Add new everify field on Award

### PR DESCRIPTION
**Pull request by db483**

### CODE REVIEW

File: src/main/java/edu/cornell/kfs/module/cg/businessobject/AwardExtendedAttribute.java [hunk 1]
```diff
@@ -88,5 +89,13 @@ public KualiDecimal getBudgetTotalAmount() {
 	public void setBudgetTotalAmount(KualiDecimal budgetTotalAmount) {
 		this.budgetTotalAmount = budgetTotalAmount;
 	}
+	
+	public boolean isEerify() {
```
Comment (jhulslander) [inline src/main/java/edu/cornell/kfs/module/cg/businessobject/AwardExtendedAttribute.java]: this is missing a V isn't it?

**Comment by jhulslander** [comment]
Code looks good aside from one missing V in a function.  I suspect CI being down is causing the unit test faiures

**Comment by db483** [comment]
Thank you for catching that, I accidentally deleted that while messing with formatting code.

**Comment by bhutchinson** [comment]
Since the unit test failures are because the CI env is down, and it's not coming back up quickly, I think it's safe to merge this PR once you are happy with the code changes.

**Comment by jhulslander** [comment]
Code looks good now, I shall merge

**Pull request merged**